eis/vocabs/readme.md

```diff
@@ -0,0 +1,19 @@
+Repository for redirecting to videogame extension vocabularies
+===================
+
+Repository created to store the redirections of the the vocabulary for AutomationML
+
+At the moment we have three games extensions:
+
+/eis/vocabs/aml: 
+
+Html: https://github.com/i40-Tools/vocabularies/tree/master/AutomationML/amlDocs/
```

Comment:
Not a comment for this file, but I did notice you were missing some markdown backtick quotes in the readme at this link so the file tree layout diagram there looks wrong.  And I could nitpick that "HTML" here should be all caps. :-)